gratipay/models/participant/email.py

```diff
@@ -106,6 +103,16 @@ def validate_email_verification_request(self, c, email, *packages):
         if not all(email in p.emails for p in packages):
             raise EmailNotOnFile()
 
+        unverified_email = c.one("""
+            SELECT address
```

Comment:
> A user could repeatedly add an email + remove it to circumvent this restriction, no? 

Eureka! Turns out it makes the most UX sense to disallow removing emails when verification is pending, to properly communicate the reality that the pending verification still exists. But, if we truly disallow removing emails when verification is pending, then we'll have closed the repeatedly-add-and-remove loophole. Violà!

Proceeding on that basis in #4579 ...